src/backend/distributed/commands/call.c

```diff
@@ -101,51 +104,33 @@ CallFuncExprRemotely(CallStmt *callStmt, DistObjectCacheEntry *procedure,
 
 	CitusTableCacheEntry *distTable = GetCitusTableCacheEntry(colocatedRelationId);
 	Var *partitionColumn = distTable->partitionColumn;
+	bool colocationWithReferenceTable = false;
```

Comment:
If the user provides a colocate_with parameter during distribution, we use that information when deciding on which placement to delegate the call to. In that sense, this is actually more than a colocation, and actually a hint that user provided during `create_distributed_function()` call

Reply:
Yeah, the naming seems a bit different here since a reference table exists on every node, I would expect anything to be colocated with a reference table. Could be nice to remove this ambiguity somehow in the future.

Reply:
I could not find a better way to rename this. I think it's ok for now.